rasa/shared/core/events.py

```diff
@@ -1404,7 +1439,7 @@ def __repr__(self) -> Text:
 
     def __str__(self) -> Text:
         """Returns event as human readable string."""
-        return self.action_name or self.action_text
+        return f"ActionExecuted({self.action_name or self.action_text})"
```

Comment:
why? Pretty sure we are using `str` to make it a human readable version (e.g. in the story validator). I think this rather fits `__repr__`

Reply:
just to make it consistent with the way we present `UserUttered`, explicitly having the name of event make it more human readable

Reply:
if you look at other events, literally all of them have their class names in `str`

Reply:
It makes it easier to debug (--> `__repr__`), not necessarily easier to read. E.g. in the story validator there is absolutely no reason to prefix this with `ActionExecuted`. I think it makes sense to prefix certain events with their class name (e.g. ones which usually don't show up in stories like `LoopInterrupted`, but I don't think that makes sense for actions (nor for `UserUttered`). 

If you absolutely want to change this back, you have to make sure that you don't introduce regression (e.g. in the way how `rasa data validate stories` warns about clashing actions).

Reply:
then `str` is wrongly used. It should be consistent along various events and independent of the event itself. I'll change it back in order not to postpone this PR

Reply:
ready to merge?